boschshcpy/models_impl.py

```diff
@@ -1350,7 +1348,6 @@ def child_lock(self, state: bool):
     
     def update(self):
         self._powerswitch_service.short_poll()
-        self._powerswitchprogram_service.short_poll()
         self._communicationquality_service.short_poll()
         self._childprotection_service.short_poll()
         super().update()
```

Comment:
You can remove the `update()` method, this is part of parent class `SHCDevice`.

Reply:
The whole update() method? All other models define an update method?

Reply:
In the current master, only the base class provides the update() method.

Reply:
Ah, good idea. Removed.